Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -0,0 +1,371 @@
+.. code:: shell-session
+
+   $ openshift-install create install-config --dir "${CLUSTER_NAME}"
+   ? SSH Public Key ~/.ssh/id_rsa.pub
+   ? Platform aws
+   INFO Credentials loaded from default AWS environment variables 
+   ? Region eu-west-1
+   ? Base Domain openshift-test-1.cilium.rocks
+   ? Cluster Name cluster-1
+   ? Pull Secret [? for help] **********************************
+
+And set ``networkType: Cilium``:
+
+.. code:: bash
+
+   sed -i 's/networkType:\ OVNKubernetes/networkType:\ Cilium/' "${CLUSTER_NAME}/install-config.yaml"
+
+Resulting configuration will look like this:
+
+.. code:: yaml
+
+   apiVersion: v1
+   baseDomain: ilya-openshift-test-1.cilium.rocks
+   compute:
+   - architecture: amd64
+     hyperthreading: Enabled
+     name: worker
+     platform: {}
+     replicas: 3
+   controlPlane:
+     architecture: amd64
+     hyperthreading: Enabled
+     name: master
+     platform: {}
+     replicas: 3
+   metadata:
+     creationTimestamp: null
+     name: cluster-1
+   networking:
+     clusterNetwork:
+     - cidr: 10.128.0.0/14
+       hostPrefix: 23
+     machineNetwork:
+     - cidr: 10.0.0.0/16
+     networkType: Cilium
+     serviceNetwork:
+     - 172.30.0.0/16
+   platform:
+     aws:
+       region: eu-west-1
+   publish: External
+   pullSecret: '{"auths":{"fake":{"auth": "bar"}}}'
+   sshKey: |
+     ssh-rsa <REDACTED>
+
+You may wish to make a few changes, e.g. increase the number of nodes. If you do change any of the CIDRs,
+you will need to make sure that Helm values used below reflect those changes. Namely - ``clusterNetwork``
+should match ``clusterPoolIPv4PodCIDR`` & ``clusterPoolIPv4MaskSize``. Also make sure that the ``clusterNetwork``
+does not conflict with ``machineNetwork`` (which represents the VPC CIDR in AWS).
+
+Next, generate OpenShift manifests:
+
+.. code:: bash
+
+   openshift-install create manifests --dir "${CLUSTER_NAME}"
+
+Now, define ``cilium`` namespace:
+
+.. code:: bash
+
+   cat << EOF > "${CLUSTER_NAME}/manifests/cluster-network-03-cilium-namespace.yaml"
+   apiVersion: v1
+   kind: Namespace
+   metadata:
+     name: cilium
+     annotations:
+       # node selector is required to make cilium-operator run on control plane nodes
+       openshift.io/node-selector: ""
+     labels:
+       name: cilium
+       # run level sets priority for Cilium to be deployed prior to other components
+       openshift.io/run-level: "0"
+       # enable cluster logging for Cilium namespace
+       openshift.io/cluster-logging: "true"
+       # enable cluster monitoring for Cilium namespace
+       openshift.io/cluster-monitoring: "true"
+   EOF
+
+.. note::
+
+   First, make sure you have Helm 3 `installed <https://helm.sh/docs/intro/install/>`_.
+
+Setup Helm repository:
+
+.. code:: bash
+
+   helm repo add cilium --force-update https://helm.cilium.io/
+
+
+Next, render Cilium manifest:
+
+.. note::
+
+   Version ``1.8-dev`` has bug fixes for OpenShift, those fixes will get included in 1.8.5.
```

Comment:
Shouldn't we just recommend here that users at least Cilium 1.8.5?

Reply:
Due to popular demand the intention is to give a preview to folks ASAP, hence it's even being backported :)

Reply:
It's a bit as if we made an RC for 1.8.5, but it turns out we don't do RCs for patch releases. This is per discussion with @joestringer and @aanm.